templates/fine-tune-llm_v2/end-to-end-examples/fine-tune-function-calling/README.md

```diff
@@ -464,28 +460,6 @@ eval_ds_gpt = get_evaluation_dataset(test_ds, TOOL_CALL_TAGS, TOOL_RESULT_TAGS,
-	<span style="color: green;">user: </span>Can you tell me about the movie &quot;Inception&quot;?
-	<span style="color: blue;">assistant: 
-		content: </span>None
-		<span style="color: blue;">tool_calls: </span>[{&#x27;function&#x27;: {&#x27;arguments&#x27;: {&#x27;title&#x27;: &#x27;Inception&#x27;}, &#x27;name&#x27;: &#x27;get_movie_info&#x27;}, &#x27;type&#x27;: &#x27;function&#x27;}]
-	<span style="color: green;">user: </span>[TOOL_RESULT] {&quot;name&quot;: &quot;get_movie_info&quot;, &quot;content&quot;: &quot;{\&quot;title\&quot;: \&quot;Inception\&quot;, \&quot;year\&quot;: 2010, \&quot;director\&quot;: \&quot;Christopher Nolan\&quot;, \&quot;genre\&quot;: [\&quot;Action\&quot;, \&quot;Adventure\&quot;, \&quot;Sci-Fi\&quot;], \&quot;plot\&quot;: \&quot;A thief who steals corporate secrets through the use of dream-sharing technology is given the inverse task of planting an idea into the mind of a CEO.\&quot;}&quot;, &quot;tool_call_id&quot;: &quot;call_1&quot;} [/TOOL_RESULT]
-	<span style="color: blue;">assistant: 
-		content: </span>The movie &quot;Inception&quot; was released in 2010. It was directed by Christopher Nolan and falls under the genres of Action, Adventure, and Sci-Fi. The plot revolves around a thief who steals corporate secrets through the use of dream-sharing technology and is given the inverse task of planting an idea into the mind of a CEO. 
-		<span style="color: blue;">tool_calls: </span>None
-	<span style="color: green;">user: </span>What about the movie &quot;The Godfather&quot;?
-	<span style="color: blue;">assistant: 
-		content: </span>None
-		<span style="color: blue;">tool_calls: </span>[{&#x27;function&#x27;: {&#x27;arguments&#x27;: {&#x27;title&#x27;: &#x27;The Godfather&#x27;}, &#x27;name&#x27;: &#x27;get_movie_info&#x27;}, &#x27;type&#x27;: &#x27;function&#x27;}]
-	<span style="color: green;">user: </span>[TOOL_RESULT] {&quot;name&quot;: &quot;get_movie_info&quot;, &quot;content&quot;: &quot;{\&quot;title\&quot;: \&quot;The Godfather\&quot;, \&quot;year\&quot;: 1972, \&quot;director\&quot;: \&quot;Francis Ford Coppola\&quot;, \&quot;genre\&quot;: [\&quot;Crime\&quot;, \&quot;Drama\&quot;], \&quot;plot\&quot;: \&quot;The aging patriarch of an organized crime dynasty transfers control of his clandestine empire to his reluctant son.\&quot;}&quot;, &quot;tool_call_id&quot;: &quot;call_1&quot;} [/TOOL_RESULT]
-	<span style="color: blue;">assistant: 
-		content: </span>&quot;The Godfather&quot; was released in 1972 and was directed by Francis Ford Coppola. It is a Crime and Drama movie. The plot is about the aging patriarch of an organized crime dynasty who transfers control of his clandestine empire to his reluctant son. 
-		<span style="color: blue;">tool_calls: </span>None
-</pre>
-
-
```

Comment:
Why was the print deleted? 

For me it helped understand what all the talk was regarding evaluation strategy. Example is not in an intuitive standard but it added to the flow.

Reply:
I don't know why these diffs are shown. I have made sure all 3 are printed and look good on the readme and ipynb.

Reply:
This was an example from the processed test dataset:

Before;
![image](https://github.com/anyscale/templates/assets/39546518/4d34cce5-2020-42b5-8119-55544279d604)


Now it looks like:
![image](https://github.com/anyscale/templates/assets/39546518/dfbdc968-1ff5-463c-8a77-143411f7da50)


I wanted to show how the the test datasets got formatted for evaluation. It's not a big deal so I'm fine if this doesn't show up ( I would prefer if we just remove the pprint line here tho).

Reply:
oh I missed this one. Will fix in the next PR.